cityofaustin/atd-moped
# Adds more fields to component view

- https://github.com/cityofaustin/atd-data-tech/issues/15349

### Steps for testing

Using an editor and admin login locally:
1. Test that the project list loads. Toggle on the **Status update** and **Partners** fields. 
2. Add a status update to a project, then confirm the status update appears in the **Status update** column of the project list.
3. Add a component to a project that you know is within any council district. Use your SQL client to inspect that the `component_arcgis_online_view.is_within_city_limits` column is `TRUE`. 
4. ☝️ Edit the component by moving its features to be outside of any council district. Verify that `is_within_city_limits` changes to `FALSE`.
5. Create a line-based component (e.g., Bike Lane) that is roughly a mile long. Use Google Maps (or what have you) to measure the distance of the component you created. Use your SQL client to verify the distances matches the value of `length_miles_total` in the DB.
